src/assets/locales/en.json

```diff
@@ -172,6 +172,8 @@
     "subText": "submit",
```

Comment:
pls change the firsts letters to upper case on this file and ru.json

Reply:
I cannot change all the buttons and therefore I will not change any

Reply:
even the ones you created?

Reply:
done all buttons with firsts letters to uppercase